graph/util/Encoding.java

```diff
@@ -164,11 +166,12 @@ public boolean isBackward() {
         VERTEX_ROLE(180, PrefixType.THING),
         STRUCTURE_RULE(190, PrefixType.RULE);
 
+
         private final byte key;
         private final PrefixType type;
 
         Prefix(int key, PrefixType type) {
-            this.key = (byte) (key & 0xff);
+            this.key = checkedCastUnsigned(key);
```

Comment:
with these new checked cast helpers, we can just specify if we want to use the range -128 to 127 or 0-255 :D